07.bitstream.semantics.md

```diff
@@ -603,6 +603,13 @@ entity identified by the ITU-T T.35 country code and terminal provider code.
 
 #### Metadata scalability semantics
 
+**Note:** The scalability metadata OBU is intended for use by intermediate processing entities that may 
+perform selective layer elimination.  Its presence allows these entities to know the structure of the 
+coded video bitstream without having to decode individual frames. 
+{:.alert .alert-info }
+
+Scalability metadata should be placed between the first sequence header and the first frame header of a coded video sequence. The information present in a scalability metadata OBU applies to the entire coded video sequence it is contained. Redundant copies of a scalability metadata OBU may occur in any temporal unit of a coded video sequence.
```

Comment:
Perhaps "it is contained." should be "it is contained in."?

This reads a bit like all bitstreams should contain scalability metadata.  Perhaps start the sentence with "If scalability metadata is present it should be ..."

Reply:
Thanks Peter. All valid points. Stephen repeated the first one, and added one more. I will produce a revision that integrates them.